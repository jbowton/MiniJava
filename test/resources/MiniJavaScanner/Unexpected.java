// unexpected charcater first
^ = 8;
//unexpected character 2nd
a = @;
// Multiple Unexpected characters
#$%&:~