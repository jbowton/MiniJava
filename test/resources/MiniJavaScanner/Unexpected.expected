<UNEXPECTED(^)> BECOMES INTLITERAL(8) SEMICOLON ID(a) BECOMES <UNEXPECTED(@)> SEMICOLON <UNEXPECTED(#)> <UNEXPECTED($)> <UNEXPECTED(%)> <UNEXPECTED(&)> <UNEXPECTED(:)> <UNEXPECTED(~)>